core/edit/edit_command_modifiers.py

```diff
@@ -64,12 +64,21 @@ def edit_modifier(m) -> EditModifier:
 }
 
 
-def run_modifier_callback(modifier: EditModifier):
-    modifier_type = modifier.type
-    if modifier_type not in modifier_dictionary:
-        raise ValueError(f"Unknown edit modifier: {modifier_type}")
-
-    count = modifier.count
-    modifier = modifier_dictionary[modifier_type]
-    for i in range(1, count + 1):
-        modifier.callback()
+@mod.action_class
+class Actions:
+    def run_modifier_callback(modifier: EditModifier):
+        """
+        Run a callback that selects or prepares text ready to apply and edit action.
```

Comment:
```suggestion
        Run a callback that selects or prepares text ready to apply an edit action.
```
Please add edit to this one as well.